include/extractor/scripting_environment_sol2.hpp

```diff
@@ -1,33 +1,30 @@
-#ifndef SCRIPTING_ENVIRONMENT_LUA_HPP
-#define SCRIPTING_ENVIRONMENT_LUA_HPP
-
-#include "extractor/scripting_environment.hpp"
+#ifndef SCRIPTING_ENVIRONMENT_SOL2_HPP
```

Comment:
Hm why did we rename this file? Sure the library that we use for executing lua changed but this is still for executing lua code.